charts/nexus3/values.yaml

```
@@ -134,6 +138,11 @@ caCerts:
   enabled: false
   secret:
 
+# if secret is provided, the license will be configured via properties, otherwise a license can still be entered manually via UI
+license:
+  enabled: false
+  secret:
```

```suggestion
  secret:
  key: nexus.license
```
The secret should have a custom key.

---

will implement this